src/components/sendV2/form/form.js

```diff
@@ -85,23 +85,22 @@ class Form extends React.Component {
 
   ifDataFromUrl() {
     const { fields = {} } = this.props;
-
     if (fields.recipient.address !== '' || fields.amount.value !== '' || fields.reference.value !== '') {
       this.setState(prevState => ({
         fields: {
           ...prevState.fields,
```

Comment:
isn't better to just use the `this.state` here?

Reply:
this is part of the solution I had to change it to work based on how the validation is being made so yes was intentional

Reply:
👌